### PR TITLE
[docs] Introduce a changelog for v0.23.1

### PR DESCRIPTION
**Pull request by VasilyStrelyaev**
\cc @DevExpress/testcafe-docs @miherlosev

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,58 @@
...
+
+```js
+runner.run({ disableTestSyntaxValidation: true })
+```
+
+### Bug Fixes
+
+* The `touchstart` event now correctly records touches ([#2856](https://github.com/DevExpress/testcafe/issues/2856))
+* Google Chrome now closes correctly after tests are finished ([#2860](https://github.com/DevExpress/testcafe/issues/2860))
+* TestCafe no longer throws errors in the console ([testcafe-hammerhead/#1769](https://github.com/DevExpress/testcafe-hammerhead/issues/1769))
```
Comment (kirovboris) [inline CHANGELOG.md]: Probably you want to say TestCafe no longer throws errors in the following cases:...? 
Or `Fixed mechanism of filtration the internal attributes` / `Fixed testcafe-hammerhead/testcafe proxy errors`?
I mean, if i were a customer I'd take this message literally, 
`TestCafe no longer throws errors in the console` - What does it mean, does TestCafe never throws an error in console?

/cc @miherlosev

Comment (miherlosev) [reply CHANGELOG.md]: Internal attributes and node changes now does not provoke MutationObserver notifications.

Comment (VasilyStrelyaev) [reply CHANGELOG.md]: Yeah, I forgot, I wanted to ask about this.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,58 @@
...
+In the API, use the [disableTestSyntaxValidation](https://devexpress.github.io/testcafe/documentation/using-testcafe/programming-interface/runner.html#run) option.
+
+```js
+runner.run({ disableTestSyntaxValidation: true })
+```
+
+### Bug Fixes
+
+* Touch events are now simulated with correct touch properties (`touches`, `targetTouches`, `changedTouches`) ([#2856](https://github.com/DevExpress/testcafe/issues/2856))
+* Google Chrome now closes correctly after tests are finished ([#2860](https://github.com/DevExpress/testcafe/issues/2860))
```
Comment (AndreyBelym) [inline CHANGELOG.md]: I think we should note that the problem was on macOS systems only.

**Pull request merged**